modules/dreamview/backend/simulation_world/simulation_world_service.h

```diff
@@ -143,9 +114,8 @@ class SimulationWorldService {
       return;
     }
 
-    adapter->SetCallback(
-        std::bind(&internal::UpdateSimulationWorld<AdapterType>,
-                  std::placeholders::_1, &world_));
+    adapter->SetCallback([this](typename AdapterType::DataType d)
+        { return SimulationWorldService::UpdateSimulationWorld(d);});
```

Comment:
if written in this way, the build will generate error as UpdateSimulationWorld(dataType) is overloaded